doc/user-manual.md

```diff
@@ -14,7 +14,7 @@
     Notes:
     1. It is like the [Azure VM Series](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes-gpu) or [GCP Machine Types](https://cloud.google.com/compute/docs/machine-types).
     2. Currently, the `skuTypes` is not directly used by HivedScheduler, but it is used by [OpenPAI RestServer](https://github.com/microsoft/pai/tree/master/src/rest-server) to setup proportional Pod resource requests and limits. So, if you are not using with [OpenPAI RestServer](https://github.com/microsoft/pai/tree/master/src/rest-server), you can skip to config it.
```

Comment:
Is that true? @yqwang-ms 
```
Currently, the `skuTypes` is not directly used by HivedScheduler, but it is used by [OpenPAI RestServer](https://github.com/microsoft/pai/tree/master/src/rest-server) to setup proportional Pod resource requests and limits. So, if you are not using with [OpenPAI RestServer](https://github.com/microsoft/pai/tree/master/src/rest-server), you can skip to config it.
```

PodSchedulingSpec has a field of gpuType (skuType) to choose certain cell chains for a pod.
```
type PodSchedulingSpec struct {
	VirtualCluster          VirtualClusterName `yaml:"virtualCluster"`
	Priority                int32              `yaml:"priority"`
	PinnedCellId            PinnedCellId       `yaml:"pinnedCellId"`
	GpuType                 string             `yaml:"gpuType"`
	GpuNumber               int32              `yaml:"gpuNumber"`
	GangReleaseEnable       bool               `yaml:"gangReleaseEnable"`
	LazyPreemptionEnable    bool               `yaml:"lazyPreemptionEnable"`
	IgnoreK8sSuggestedNodes bool               `yaml:"ignoreK8sSuggestedNodes"`
	AffinityGroup           *AffinityGroupSpec `yaml:"affinityGroup"`
}
```

Reply:
It's true. The gpuType in PodSchedulingSpec is the leaf cell type, which is inferred from cell chain in cellTypes config instead of gpuType**s** config.